doc/man3/SSL_CONF_cmd.pod

```diff
@@ -441,6 +452,15 @@ middleboxes that do not understand TLSv1.3 will not drop the connection. This
 option is set by default. A future version of OpenSSL may not set this by
 default. Equivalent to B<SSL_OP_ENABLE_MIDDLEBOX_COMPAT>.
 
+B<AntiReplay>: If set then OpenSSL will automatically detect if a session ticket
+has been used more than once, TLSv1.3 has been neotiated and early data is
```

Comment:
(typo+comma again)

Reply:
( I gotta say that I appreciate the consistency of the typo ;-) )

Reply:
Fixed.